client/src/components/Dialog/PostDialog/PostDialog.js

```diff
@@ -5,14 +5,14 @@ import DialogActions from '@material-ui/core/DialogActions';
 import DialogContent from '@material-ui/core/DialogContent';
 import DialogTitle from '@material-ui/core/DialogTitle';
 import { Chip } from '@material-ui/core';
-import CircularProgress from '@material-ui/core/CircularProgress';
+// import CircularProgress from '@material-ui/core/CircularProgress';
```

Comment:
don't comment out code like this, delete it instead

Reply:
yeah I noticed the validation stuff too. I'm working on integrating the boards and posts right now, but I can tackle the validation stuff later today